contracts/RequestResponseConsumer.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
-import { RequestResponseConsumerFulfillUint128 } from "@bisonai/orakl-contracts/src/v0.1/RequestResponseConsumerFulfill.sol";
-import { RequestResponseConsumerBase } from "@bisonai/orakl-contracts/src/v0.1/RequestResponseConsumerBase.sol";
-import { Orakl } from "@bisonai/orakl-contracts/src/v0.1/libraries/Orakl.sol";
+import {RequestResponseConsumerFulfillUint128} from "@bisonai/orakl-contracts/src/v0.1/RequestResponseConsumerFulfill.sol";
```

Comment:
@bayram98 Can you please also add a withdrawTemporary to this mock contract? Without that, consumer could not get money back if they request data through temporary account is not fulfilled.